README.md

```diff
@@ -99,6 +99,15 @@ Scripts for updating and debugging Kafka can be found [here](https://github.com/
 | DEFAULT_LIMIT                      | `20`                                                                                                                                                                                                      | Default limit for pagination                                                                         |
 | DEFAULT_OFFSET                     | `0`                                                                                                                                                                                                       | Default offset for pagination                                                                        |
 |                                    |                                                                                                                                                                                                           |                                                                                                      |
+| OTEL_BATCH_TIMEOUT                 | 
```

Comment:
Small point but the column seperators are not formatted the same as the lines before (number of spaces) so the table displays poorly in raw mode.

Reply:
The spacing has now been updated.